jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/identity/CustomUserGroupCallbackImpl.java

```diff
@@ -0,0 +1,152 @@
+package org.jbpm.services.task.identity;
+
+import java.io.FileInputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.Set;
+
+import org.kie.api.task.UserGroupCallback;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Custom User Group Callback for Human Tasks
+ */
+public class CustomUserGroupCallbackImpl implements UserGroupCallback {
```

Comment:
in general this callback has very limited value and frankly speaking no value for production deployments, note that application-users.properties and application-roles.properties are EAP/WidFly specific and we already have JBoss like user group callback - https://github.com/kiegroup/jbpm/blob/master/jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/identity/JBossUserGroupCallbackImpl.java.
So I opt at removing this one completely
Here you can find all callbacks we ship out of the box https://github.com/kiegroup/jbpm/blob/master/jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/identity

Reply:
Done